server/routes/billRoute.js

```diff
@@ -102,4 +102,30 @@ router.post(
 	}
 )
 
+// @route   GET app/bill/:userid
+// @desc    Get all bills for under a user with userid.
+// @access  Private
+// router.get('/:userid', [auth], async (req, res) => {
+// 	const { userid } = req.params;
+// 	console.log(userid);
+
+// 	try {
+// 		const user = await User.findOne({__id: userid});
+// 		res.json({user});	
+// 	} catch (error) {
+// 		res.json({error});
+// 	}
+// })
+
+router.get('/', [auth], async (req, res) => {
+	const userid = req.query.userid;
+	
+	try {
+		const bills = await Bill.find({hostID: userid});
+		res.json({bills});	
+	} catch (error) {
+		res.json({error});
+	}
+})
+
```

Comment:
Added route to get all the bills of an user.

Reply:
why did you comment out my code?

Reply:
That's not your code. I wrote both routes but the one on the top didn't work and I forgot to take it out.